app/javascript/schools/courses/components/course_editor/CourseEditor__Form.re

```diff
@@ -72,12 +72,12 @@ module UpdateCourseQuery = [%graphql
 ];
 
 let updateName = (send, name) => {
-  let hasError = name |> String.length < 2;
+  let hasError = name |> String.trim |>String.length < 2;
```

Comment:
It appears that your editor is misconfigured, and isn't triggering `refmt`. All of these changes look _slightly_ different with the correct formatting.

I'm hoping that you're using VSCode with the recommended [reason-vscode](https://marketplace.visualstudio.com/items?itemName=jaredly.reason-vscode) plugin. With the right settings, VSCode should be reformatting Reason files when you save changes. Here's [the settings that I use](https://github.com/harigopal/config/blob/master/vscode.json) with VSCode.